scripts/scr_add_man/scr_add_man.gml

```diff
@@ -74,8 +68,8 @@ function scr_add_man(argument0, argument1, argument2, argument3, argument4, argu
 	                if (loaded=0){
```

Comment:
this will always be true, since 4 lines above we have already checked taht (loaded == 0)